api/index.js

```diff
@@ -14,4 +14,8 @@ apiRouter.get('/health', (req, res, next) => {
 
 // place your routers here
 
+//ROUTER: /api/cart
+const cartRouter = require('./cart');
+Router.use("/cart", cartRouter);
```

Comment:
Youll want to reference apiRouter here.

```suggestion
apiRouter.use("/cart", cartRouter);
```